srcpkgs/python3-pywlroots/template

```diff
@@ -36,3 +35,9 @@ pre_build() {
 post_install() {
 	vlicense LICENSE
 }
+
+python3-pywlroots-0.16_package() {
+	build_style=meta
+	short_desc+=" - transitional dummy package"
+	depends="${sourcepkg}>=${version}_${revision}"
+}
```

Comment:
not needed, imho

Reply:
Because Qtile require python3-pywlroots and the only one so no need for a transition from python3-pywlroots-0.16 to python3-pywlroots?
It was @ahesford who mention it so that's why I added it, I can remove it if that is the consensus.

Reply:
I don't have a strong opinion here. I guess having the new package replace the old is sufficient.

Reply:
It's now removed.